packages-next/fields/src/types/select/tests/test-fixtures.ts

```diff
@@ -1,14 +1,16 @@
-import Text from '../Text';
-import Select from './';
+// @ts-ignore
+import { Text, Select } from '@keystone-next/fields-legacy';
+
+type MatrixValue = 'enum' | 'string' | 'integer';
```

Comment:
(here + in all the other places)
```suggestion
type MatrixValue = typeof testMatrix[number];
```